src/_base/helm/app/values.yaml.twig

```diff
@@ -3,8 +3,11 @@ appVersion: {{ @('app.version') | json_encode | raw }}
 feature: {{ to_nice_yaml(@('helm.feature'), 2, 2) | raw }}
 
 ingress:
+  annotations: {{ to_nice_yaml(
+    (@('pipeline.base.istio.enabled') ? @('pipeline.base.istio.ingress.annotations') : @('pipeline.base.services.nginx.ingress.annotations')),
+    2, 4) | raw }}
```

Comment:
why would we need to specify both sets of annotations conditionally? I can only see if our harness was to set some up ourselves

Reply:
Whilst I doubt the istio ingress controller would complain, an annotation like below doesn't make sense for istio's ingress:
```
nginx.ingress.kubernetes.io/proxy-body-size: 100m
```

Reply:
sorry if not clear what I mean, why not @('pipeline.base.ingress.annotations')

Reply:
do we set that in the harness, or are we considering it for the future to?

Reply:
The annotations are customisable per project/per cluster and empty by default

Reply:
ok, so per project, where we are using both for different environments, and would be setting these in workspace.yml, rather than the cluster repo?

Reply:
I guess it wouldn't have both on a project - it would be too inconsistent between environments. Will simplify tomorrow :+1:

Reply:
Updated :)